applications/Bifrost-PLO-Liquidity.md

```diff
@@ -0,0 +1,153 @@
+
+- Investers: DOT, KSM token holders, parachain users and investment institutions are all potential investors in the parachain slot auction.
+- Parachain: Need to auction parachain slots, access polkadot or kusama relay projects.
+- Bifrost: Through decentralized provision of vsDOT or vsKSM derivatives, the middle layer protocol that unlocks the liquidity of the parachain slot auction。
+- Polkadot/Kusama: Relay chain that has parachain slots and released for auction winners.
+
+### Runtime modules
+To implement the above mechanism, we will add the following modules to Substrate runtime of Bifrost:
+1. Prepare PLO module
+It contains the following features:
+    - Parachain Registration: Registration of the parachain’s information and PLO-related arguments on Polkadot-Relaychain.
+    - Investors DOTs Deposit: Investors Give Bifrost the capabitlity of bounding their DOTs to support Parachain PLO, and get vsDOT/vsKSM.
+2. Execute PLO module
+It contains the following features:
+    - Bounding DOTs: Bounding DOTs for parachains on Polkadot relay chain to support their PLO.
+    - Redeeming DOTs: When the PLO fails or slot lease expires, contributors will receive the returned DOTs.
+2. Asset module
+It contains the following features:
+    - Record Certifications: The certifications represent the corresponding quantity of user's contribution, parachains will reward users who hold contributions. Bifrost records these contribution holders, in order that keep their right to redeem 1:1 KSM/DOT after the fund being retirememt.
+    - DOT-vsDOT or KSM-vsKSM Conversion: When fund being retirement, vsDOT/vsKSM should be converted to DOT/KSM.
+
+### Workflow
+- Investors participate in the parachain slot auction process
+![171611737050_ pic](https://user-images.githubusercontent.com/72777624/105987911-e5880b00-60d9-11eb-8557-0f46c8c7f121.jpg)
+
+- Investors use Bifrost to participate in the parachain card slot auction process with liquidity
+![181611737079_ pic_hd](https://user-images.githubusercontent.com/72777624/105987978-ff295280-60d9-11eb-8adc-23058f51b605.jpg)
+
+### Security
+Investors use XCMP to transfer DOTs to the Bifrost platform, so the security of user funds is guaranteed by XCMP. Parachain projects that require PLO also use XCMP to deposit rewards asset on the Bifrost platform. For example: Alice transfer her DOT in relay chain to the Bifrost parachain through the XCMP cross-chain function, and then ALice can call the function provided by the Bifrost platform to bind ParachainA. Her rewards will be released linearly in time. The unreleased part is always locked and no one can embezzle it. It is as safe as being locked in the parachain itself. Parachain projects can also choose to deposit rewards gradually on the Bifrost platform in multi-batches.
```

Comment:
What rewards do you mean?

Reply:
These rewards are issued by parachain candidates for investors, usually in the form of tokenized assets or parachain-coin, which can be transferred in the Polkadot ecosystem through the XCMP protocol.

Reply:
> In addition to @jonasW3F's comments, could you elaborate on the status of the previous grant? Are you still actively working on it?
> We usually don't support parallel grants for a team, but rather require them to complete one project before agreeing on a follow-up grant.

Yes, we do, we've updated the previous grant, feel free to check it.